app/controllers/api/v0/highlights_controller.rb

```diff
@@ -48,6 +48,56 @@ def create
     render json: response_binding, status: :created
   end
 
+  swagger_path '/highlights' do
+    operation :get do
+      key :summary, 'Get the highlight(s)'
+      key :description, 'Get the highlight(s) for the owner.'
+      key :operationId, 'getHighlights'
+      key :produces, [
+        'application/json'
+      ]
+      key :tags, [
+        'Highlights'
+      ]
+      # security do
+      #   key :api_id, []
+      # end
+      parameter do
+        key :name, :get_highlights
+        key :in, :query
+        key :description, 'The get highlight query parameters'
+        key :required, true
+        schema do
+          key :'$ref', :GetHighlights
+        end
+      end
+      response 201 do
+        key :description, 'Returns the desired highlight(s).'
+        schema do
+          key :'$ref', :Highlights
+        end
+      end
+      extend Api::V0::SwaggerResponses::AuthenticationError
+      extend Api::V0::SwaggerResponses::ForbiddenError
```

Comment:
Forbidden (403) not possible since we aren't querying specific highlights belonging to someone else, just those belonging to the user.

Reply:
good point!